core/trino-main/src/main/java/io/trino/SystemSessionProperties.java

```diff
@@ -162,7 +162,12 @@
     public static final String RETRY_DELAY_SCALE_FACTOR = "retry_delay_scale_factor";
     public static final String LEGACY_MATERIALIZED_VIEW_GRACE_PERIOD = "legacy_materialized_view_grace_period";
     public static final String HIDE_INACCESSIBLE_COLUMNS = "hide_inaccessible_columns";
-    public static final String FAULT_TOLERANT_EXECUTION_TARGET_TASK_INPUT_SIZE = "fault_tolerant_execution_target_task_input_size";
+    public static final String FAULT_TOLERANT_EXECUTION_ADAPTIVE_TASK_SIZING_STEP_SIZE = "fault_tolerant_execution_adaptive_task_sizing_step_size";
+    public static final String FAULT_TOLERANT_EXECUTION_ADAPTIVE_TASK_SIZING_GROWTH_FACTOR = "fault_tolerant_execution_adaptive_task_sizing_growth_factor";
```

Comment:
Hmm - I would have those separate for writer and arbitrary too. For arbitrary we probalby want 2x less frequently. And for writers some smaller factor but more often for more smooth growth.